java/src/com.zeroc.ice/src/main/java/com/zeroc/Ice/ConnectFailedException.java

```diff
@@ -4,14 +4,23 @@
 
 /** This exception indicates connection failures. */
 public class ConnectFailedException extends SocketException {
+    /**
+     * Constructs a new {@code ConnectFailedException}.
```

Comment:
I don't really care, but in all the previous exceptions we went with "constructs a ..." instead of "constructs a new ...".
I think we settled on omitting the "new" for constructors, but whichever we pick, we should be consistent.
```suggestion
     * Constructs a {@code ConnectFailedException}.
```